.github/workflows/aur.yml

```diff
@@ -0,0 +1,42 @@
+name: aur
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  aur-publish:
+    name: Publish to AUR
+    runs-on: ubuntu-latest
+    container: archlinux/base
+
+    steps:
+      # see: https://github.community/t5/GitHub-Actions/How-to-get-just-the-tag-name/m-p/32167#M1027
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
```

Comment:
I've tried that but the problem is we don't check out the dotenv-linter itself so we don't have access to it's git repository - we only check out the `dotenv-linter-bin` repository from the AUR.
That's why I went with this solution.
Of course we could check out the dotenv-linter itself, but that would cost a bit of performance.

Do you think it would be worth it?

Reply:
Thank you for explaining me. I think no. Let's leave as is.